webpack/assets/javascripts/react_app/components/common/ModalProgressBar/ModalProgressBar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, ProgressBar } from 'patternfly-react';
+import { sprintf, translate as __ } from '../../../common/I18n';
+
+const ModalProgressBar = props => {
+  const { show, container, task } = props;
+  let modalTitle = null;
+  let progress = 0;
+
+  if (task) {
```

Comment:
i wonder if this should be in tasks plugin, or perhaps a different component wrapping this one?

Reply:
@iNecas what do you think?

Reply:
We plan introducing react/redux into foreman-tasks, so this code could be moved there once the infra is there. //cc @sharvit

Reply:
IMHO we can remove lines 11-14 and just receive the `modalTitle` and the `progress` as props.
Those lines should move to Katello so when consuming this component, Katello can pass the 2 relevant props instead of the whole `task` object.

That's way, we will have a more reusable `ModalProgressBar` that not only work with tasks.